src/styles/stars.scss

```diff
@@ -0,0 +1,38 @@
+.stars {
+  display: flex;
+  align-items: center;
+  height: 16px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:nth-child(5) {
+    margin-right: 0;
+  }
+}
+
+.stars--1 div:nth-child(1) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--2 div:nth-child(-n + 2) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--3 div:nth-child(-n + 3) {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars--4 div:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--5 div:nth-child(n) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
You can just list the selector using a comma and then write the same rule for different selectors. No need to repeat yourself.

Reply:
Ok Gotcha sensei 👌